dev/com.ibm.ws.security.jwt/resources/com/ibm/ws/security/jwt/internal/resources/JWTMessages.nlsprops

```diff
@@ -386,3 +386,21 @@ NESTED_JWS_REQUIRED_BUT_NOT_FOUND.useraction=Ensure that the payload of the JWE
 JWE_DECRYPTION_KEY_MISSING=CWWKS6066E: The JSON Web Encryption (JWE) token cannot be decrypted because a decryption key cannot be found. The {0} server configuration attribute is set to [{1}].
 JWE_DECRYPTION_KEY_MISSING.explanation=A decryption key is required to decrypt the JWE token. The decryption key must be present in the JWT consumer keystore, or the key must be specified in the MicroProfile Config properties.
 JWE_DECRYPTION_KEY_MISSING.useraction=Verify that the key alias that is specified in the message exists in the JWT consumer keystore. If the alias is not configured, the decryption key must be specified in the MicroProfile Config properties.
+
+# Do not translate "JSON Web Token", "JWT", "iat"
+# 0=iat claim time value (date), 1=Current time, 2=Clock skew configuration value, 3=Token age configuration value
+JWT_TOKEN_AGED=CWWKS6067E: The JSON Web Token (JWT) is not valid because the token age since issued at (''iat'') has elapsed. The ''iat'' claim time is [{0}]. The current time minus the clock skew is [{1}]. The configured clock skew is [{2}] seconds. The configured token age is [{3}] seconds.
```

Comment:
```suggestion
JWT_TOKEN_AGED=CWWKS6067E: The JSON Web Token (JWT) is not valid because the token is too old. The ''iat'' claim time is [{0}]. The current time minus the clock skew is [{1}]. The configured clock skew is [{2}] seconds. The configured token age is [{3}] seconds.
```